### PR TITLE
(partially) Revert "Vitamin buff and removal from plants + frying and sugar nerf"

### PR DESCRIPTION
**Pull request by Sniblet**
This reverts commit 92fd210fd44232854d4268817d3528f3b111345b.
https://github.com/yogstation13/Yogstation/pull/10971

# General Documentation

### Intent of your Pull Request

Makes vegetables healthy again. I don't see why their vitamin removal makes any sense from a balance perspective, and it's certainly unreasonable logically.
If you want cooks to be better than botanists for food, fine, but there's no need to do it through nerfs and to make the station completely unable to access vitamins without a cook onboard.
I don't really know or care about the cooking oil and sugar changes, I'm not reverting those

### Why is this change good for the game?

Going back for another half-dozen carrots every 5-10 minutes is not engaging spaceman gameplay
Going back for another three carrots every 15-20 minutes was much more exciting

### What should players be aware of when it comes to the changes your PR is implementing?

Vegetables are now liable to keep you full and healthy for more than 5 minutes per visit to botany

### What general grouping does this PR fall under? 
Service and nutrition

# Changelog

:cl:  
tweak: vitamins heal less, nutriment heals more, plants have vitamins again because they're not junk food
/:cl:

### CODE REVIEW

**Comment by ynot01** [comment]
botany buf

**Comment by Marmio64** [comment]
keep the oil and sugar changes.

The reasoning for removing vitamins from plants was to make prepared food better since botany just always takes over the chefs job for him. Botany really doesn't need to be something that can make the chef totally useless and there isn't any good ways to buff chef food other than just more mood buffs, which practically nobody has on. Also I never have any problems with food, even now. I eat fruits till i'm full and then pocket another one for later and I only ever have to make one or two trips back during the course of a shift. And, you need to edit the PR title so that people understand from a glance what it is reverting.

**Comment by Cartlord** [comment]
Something I'm actually surprised by, is that nobody here seems to have considered that Botany can make their own meat - either via meatwheat steak & jerky, or by using the biogen to make monkey cubes.

File: code/modules/hydroponics/grown/tea_coffee.dm [hunk 1]
```diff
@@ -83,7 +83,7 @@
 	plantname = "Coffee Robusta Bush"
 	product = /obj/item/reagent_containers/food/snacks/grown/coffee/robusta
 	mutatelist = list()
-	reagents_add = list(/datum/reagent/medicine/ephedrine = 0.1, /datum/reagent/toxin/coffeepowder = 0.1)
+	reagents_add = list(/datum/reagent/medicine/ephedrine = 0.1, /datum/reagent/consumable/nutriment/vitamin = 0.04, /datum/reagent/toxin/coffeepowder = 0.1)
```
Comment (Cartlord) [inline code/modules/hydroponics/grown/tea_coffee.dm:86]: Why would COFFEE have vitamin?

**Comment by Cartlord** [comment]
Something I'm noticing is that nearly EVERY edible plant seems to have vitamin, I'm frankly opposed to that - melons, for instance, already provide insane nutriment, why give them vitamin too?
I think it might be fine on mutated plants (not the Ambrosia chain, because bloody everyone goes for those) or weeds, but it seems ludicrously to get in your code.
I don't know if you play Botany, so I'll assume you don't know it too well. A good Botanist, or even a decent one, can make dozens of plants per minute. If those plants have vitamin, there's screw-all stopping them from just bagging the plants, grinding them, and using the CondiMaster to extract pure vitamin in HORRIFIC quantities.
I know the DNA manipulator can Extract the vitamin content gene, but at least if it's on a mutate plant (especially one people don't usually go for, like Spaceman's Trumpet or something) they need to go OUT OF THEIR WAY to get it.
Maybe put it on sweet potatoes? They're a food item, a plant you don't see much, and a mutation - so you can make vitamin, but you at least need to DO something to get it.

**Pull request closed without merging**